tests/foreman/cli/test_contentviews.py

```diff
@@ -1502,14 +1392,15 @@ def test_cv_publish_version_changes_in_target_env(self):
         # Actual assert for this test happens HERE
         # Test whether the version1 now belongs to Library
         result_version = ContentView.version_info({u'id': version1_id})
-        self.assertEqual(result_version.return_code, 0,
-                         "ContentView version was not found")
-        self.assertEqual(len(result_version.stderr), 0,
-                         "No error was expected")
+        self.assertEqual(
+            result_version.return_code, 0,
+            "ContentView version 1 was not found"
+        )
+        self.assertEqual(len(result_version.stderr), 0)
         self.assertEqual(
             result_version.stdout['lifecycle-environments'][0]['label'],
             u'Library',
-            'This version not in Library'
+            'Version 1 is not in Library'
```

Comment:
Yes, we have two versions being created on this test, and I decided to leave this message to make easier to know if the first or second version that failed.